pkg/platform/platform.go

```diff
@@ -22,6 +22,39 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/version"
 )
 
+// CommonControlPlaneCharts defines a list of control plane Helm charts to be deployed for all
+// platforms.
+var CommonControlPlaneCharts = []ControlPlaneChart{
```

Comment:
To avoid global variables, can we convert that to function please? 

Also, we now have this code: https://github.com/kinvolk/lokomotive/blob/master/cli/cmd/cluster-apply.go#L109-L121. It seems like something which should be consolidated (I also like that format better, as namespace is not repeated).